MiniBootcamp/MiniBootcamp/TweetModel.swift

```diff
@@ -0,0 +1,67 @@
+//
+//  TweetModel.swift
+//  MiniBootcamp
+//
+//  Created by Chioma Amanda Mmegwa  on 29/05/2023.
+//
+
+import UIKit
+
+struct TweetCellViewModel {
+    let userName: String
+    let profileName: String
+    let profilePictureName: String
+    let content: String
+    
+    var profilePicture: UIImage? {
+        return UIImage(named: profilePictureName)
+    }
+}
+
+// MARK: - TweetsResponse
+struct TweetsResponse: Codable {
```

Comment:
`Codable` is a type alias for the `Encodable` and `Decodable` protocols. And here I do not think that adding both protocols are necessary. Since you are only decoding, just add Decodable.